src/globus_sdk/services/auth/client/base_login_client.py

```diff
@@ -73,6 +73,10 @@ def __init__(
             f"client_id='{client_id}', type(authorizer)={type(authorizer)}"
         )
 
+    # FYI: this get_openid_configuration method is duplicated in AuthClient
+    # if this code is modified, please update that copy as well
+    # this will be resolved in SDK v4 by removing this copy and restructuring the
+    # LoginClients to contain AuthClient instances
```

Comment:
It's possible that this comment will prove incorrect when we launch v4. Perhaps just these two lines could be removed.

```suggestion
```

Reply:
I see how you're coming at this, and yes, these could be made inaccurate if we release v4 without changing things in this way.

However, I would really like to have some kind of expressed opinion in the code about which one is meant to stay in place and which should be removed, since that's tightly related to my decision to duplicate the code between the two classes (rather than trying to find some abstraction for it). And I think that declaration belongs in the code -- if we only have it in issue tracking, the information is disassociated from the files which really need this added context.

Therefore, I'd like to find a phrasing which is more open and doesn't tie us to v4 specifically, but which still expresses the same idea. Maybe...

```suggestion
    # we would like to restructure code using this method to be calling the matching
    # AuthClient method
    # for example, a future SDK version may make an AuthLoginClient contain
    # an AuthClient which it uses
```

Does that seem more on-target?

Reply:
That works for me.

Reply:
I went ahead with this, nearly verbatim. If it mattered enough to comment, then IMO it matters enough to push a fix -- it's trivial to change it like this anyway.

I'll wait for CI and then probably merge on the strength of the existing approval.